lib/src/service/media_picker_service.dart

```diff
@@ -47,4 +48,19 @@ class MediaPickerFileService implements MediaPickerService {
     await controller.setLooping(true);
     await controller.play();
   }
+
+  @override
+  Future<FilePickerResult?> pickFile(List<String> extensions) async {
```

Comment:
Rewrite to unit8list plz

Reply:
Had that first, but I need to use the file name and extension too, so that's why i rewrote it to use the file picker result. The final result that gets returned should be an uint8list